tools/astarte_export/lib/fetchdata/queries/queries.ex

```diff
@@ -0,0 +1,191 @@
+defmodule Astarte.Export.FetchData.Queries do
+  alias Xandra
+  alias Astarte.Import.LogFmtFormatter
+  alias Astarte.Core.InterfaceDescriptor
+  alias Astarte.Core.Mapping
+  require Logger
+
+  def get_connection() do
+    nodes = Application.get_env(:cqerl, :cassandra_nodes)
```

Comment:
Why do you use cqerl conf if your'e using Xandra?

Reply:
removed